lua/plugins/configs/nvimtree.lua

```diff
@@ -102,4 +65,44 @@ require("nvim-tree").setup {
       enable = true,
       update_cwd = true,
    },
+   view = {
+      allow_resize = true,
+      side = "left",
+      width = 25,
+      mappings = {
+         list = {
+            { key = { "<CR>", "o", "<2-LeftMouse>" }, cb = tree_cb "edit" },
```

Comment:
I don't know why adding default keymappings.

Reply:
@ashincoder we might change em in the future and it gets easier for the user to just look at the binds in the config if they dont know g? nvimtree keybind